style.css

```diff
@@ -0,0 +1,481 @@
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: 5%;
+}
+
+.single {
+  display: flex;
+  background-color: #f8f6f6;
+  background-image: url("./images/Chess.jpg");
+  background-size: 15%;
+  background-repeat: no-repeat;
+}
+
+.img-speaker {
+  width: 110px;
+  height: 110px;
+  padding: 15px;
+}
+
+.speeches {
+  padding: 10px;
+}
+
+.speech-title {
+  font-size: 30px;
+  margin: 0;
+}
+
+.profesion {
+  color: #ec5242;
+}
+
+.pro-break {
+  margin-right: 90%;
+}
+
+.partner-cont {
+  padding-bottom: 50px;
+  background-color: #272a31;
+}
+
+.partner-heading {
+  padding-top: 50px;
+  margin-left: 20px;
+  text-align: center;
+  color: #d3d3d3;
+  font-size: 35px;
+}
+
+.break-partner {
+  margin-left: 45%;
+  margin-right: 45%;
+  border: 1px solid #ec5242;
+  margin-bottom: 50px;
+}
+
+.partners {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 15%;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.partners2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  gap: 10%;
+  justify-content: center;
+}
+
+.partners-icon {
+  width: 100px;
+  transition: all 0.5s ease-in-out;
+}
+
+.partners-icon:hover {
+  transform: scale(1.1);
+}
+
+footer {
+  margin-top: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding-bottom: 20px;
+}
+
+.footer-img {
+  width: 20%;
+  height: auto;
+}
+
+footer > h5 {
+  font-size: 25px;
+  color: gray;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    padding: 0;
+    margin: 0;
+    font-family: "lato", sans-serif;
+  }
+
+  .main {
+    padding-left: 0;
+    padding-top: 0;
+    padding-right: 0;
+  }
+
+  header {
+    display: block;
+    background-color: #272a31;
+  }
+
+  .lists {
+    display: flex;
+    justify-content: end;
+    list-style-type: none;
+    color: #fff;
+    margin-right: 13%;
+    gap: 2.76%;
+    padding: 10px;
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .social-media {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .nav {
+    display: none;
+  }
+
+  .nav-desk {
+    display: flex;
+    justify-content: space-between;
+    padding-right: 13%;
+    padding-left: 13%;
+    align-items: center;
+    margin-top: 0;
+    background-color: #d3d3d3;
+  }
+
+  .logo-desk > img {
+    width: 90px;
+    height: 80px;
+  }
+
+  .navi-menu-desk {
+    display: flex;
+    gap: 30px;
+  }
+
+  .navi-link-desk {
+    text-decoration: none;
+    color: #272a31;
+    padding: 10px;
+  }
+
+  .home,
+  .sponsor {
+    color: #ec5242;
+  }
+
+  .sponsor {
+    border: 4px solid #ec5242;
+  }
+
+  .hello,
+  .main-header,
+  .info,
+  .time {
+    margin-left: 13%;
+    margin-right: 13%;
+  }
+
+  .hello {
+    font-size: 25px;
+    margin-top: 80px;
+  }
+
+  .main-header {
+    font-size: 45px;
+  }
+
+  .info {
+    font-size: 25px;
+    padding: 20px;
+    border: 1px solid #d3d3d3;
+  }
+
+  .interval {
+    font-size: 33px;
+  }
```

Comment:
- Here's the correct font-size: 24px. Kindly fix this as the issue is not yet resolved as requested by the previous reviewer.